### DISCUSSION TITLE
Ability to set History Graph colors

### DISCUSSION THREAD

**Discussion by aaron9060** [home-assistant/frontend#11188] [Other feature requests]
It would be nice to be able to change the default colors applied to Lovelace history graph cards.  The default colors seem somewhat arbitrary and can clash with other elements in the UI. 

Or perhaps to select from a different palette if setting each entity and potential value is too troublesome?

<img width="380" alt="history_graph" src="https://user-images.githubusercontent.com/6381270/61584016-583c2180-ab06-11e9-94cf-1ff85f212bb7.PNG">

        type: history-graph
        title: 'Lighting History'
        entities:
          - entity: light.counter_switch
            name: 'Counter'
          - entity: light.bedroom_lights
            name: 'Bedroom'
          - entity: light.office_lights
            name: 'Office'            
          - entity: light.bathroom_lights
            name: 'Bathroom'
          - entity: light.basement_lights
            name: 'Basement'
          - entity: light.utilityroom_lights
            name: 'Utility'
          - entity: light.garage_lights
            name: 'Garage'
          - entity: light.porch_switch
            name: 'Porch'
          - entity: light.deck_switch
            name: 'Deck'
          - entity: light.landscape_lights
            name: 'Landscape'

**Comment by noxhirsch** [comment]
While I don't think that this old issue will get attention anymore 😢 there are two additional issues with these colors:
1. When using `history-graph` with an `binary_sensor` for doors and windows, **closed** is red, while **open** is green which is the opposite you want, if you have these sensors for security reasons (then open doors or windows are bad and should be red).
2. When `history_graph` is used with an regular sensor, you get different colors for the same state (depending on which state comes first). So in this case, I can't see at a glance, which door is unlocked at which times, because the colors are not consistent:
![barcard-color](https://user-images.githubusercontent.com/30938717/87006189-8ce63480-c1c0-11ea-9fe2-d452333e166c.jpg)

**Comment by thomasloven** [comment]
Workaround: https://gist.github.com/thomasloven/5ee36708908569c8e168419557cefd08

**Comment by jenniferlee1818** [comment]
8% of human males and 0.6% of females of Northern European ancestry are red-green color-blind.

**Comment by ankostis** [comment]
@noxhirsch i notice that in  2021.5.5 the binary sensors for doors & windows have been fixed:
- red: `open`
- green: `closed`

but this logic breaks in motion-sensors, where:
- red: `clear`
- green: `detected`.

In my screenshot, the top 3 rows are motion-sensors, the bottom 2 window magnets:
![image](https://user-images.githubusercontent.com/501585/120864577-18b13e00-c595-11eb-8ce0-e8b8e3c940cd.png)

**Comment by ankostis** [comment]
Should i open a separate new issue about motion sensors?
